R/classify.R

```diff
@@ -31,7 +51,8 @@ subtype_labels <- c("WNT", "SHH", "G3", "G4")
 #'
 #' @examples
 #' \dontrun{
-#' # Assumes input_data_matrix is a properly-formatted sample by gene expression matrix
+#' # Assumes input_data_matrix is a properly-formatted
+#' sample by gene expression matrix with relative expression values
```

Comment:
Should this be? Genuinely having some trouble based on the diff
```suggestion
#' # sample by gene expression matrix with relative expression values
```

Reply:
Should be, thanks!! I'll double check these in general too.